### ISSUE TITLE
Eventually wrong TH_3 in traces 

### ISSUE THREAD

**Issue by StefanHri** [lake-wg/edhoc#316]
@stoprocent 

The input for TH_3 given in the traces doc [1]:
```
58203AB11700841FCE193C323911EDB317B046DCF24B9950FD624884F7F57CD98B07
5850A11822822E4879F2A41B510C1F9B5840F373A7203EFA7DF0738C360EE080171A
CA67FCB175F26D785D095B55EB1484655C39E03A3F5F9BDD87EF8C5F2EC3DFE6FBBA
49B7B4625B126F27DE301767270C
```
produces the hash value:
```
ca4915a1c4096febd9351c93ff1967b0fc7c9d72809574512709954771f2c66e
```
and not
```
77 a5 1a 1c f0 eb 60 e4 06 1b fc 27 08 87 52 9c 16 b6 f4 b3 4d 88 a2 26
e7 06 5b 60 f4 15 26 71
```

[1]: https://lake-wg.github.io/edhoc/traces-of-14/draft-ietf-lake-traces.html

**Comment by StefanHri** [comment]
Thanks for the fast response Marco! Can you share the complete raw traces?

**Comment by marco-tiloca-sics** [comment]
Except for the fixes above (to be added), the raw traces are both available at:

https://github.com/lake-wg/edhoc/blob/337615d8af2a4c11e70ab4bdf50e2e05be0c4a6a/draft-ietf-lake-traces.md

as part of PR #301

**Comment by marco-tiloca-sics** [comment]
The fixes above are not added to PR #301, see the commit https://github.com/lake-wg/edhoc/pull/301/commits/3c53df19e3614b98c4d9384e38c07fe230ab9820

**Comment by stoprocent** [comment]
@StefanHri have you looked at the ones generated by me? https://github.com/lake-wg/edhoc/pull/315/files
